core/jobs_test.py

```diff
@@ -1221,18 +1193,19 @@ def test_get_continuous_computations_info_with_existing_model(self):
         job_models.ContinuousComputationModel(
             id='StartExplorationEventCounter').put()
         continuous_computations_data = jobs.get_continuous_computations_info(
-            self.ALL_CC_MANAGERS_FOR_TESTS)
+            [StartExplorationEventCounter])
 
-        self.assertEqual(len(continuous_computations_data), 1)
-        continuous_computation_data = continuous_computations_data[0]
+        expected_continuous_computations_data = {
+            'computation_type': 'StartExplorationEventCounter',
+            'status_code': 'idle',
+            'is_startable': True,
+            'is_stoppable': False
+        }
 
-        self.assertEqual(
-            continuous_computation_data['computation_type'],
-            'StartExplorationEventCounter')
-        self.assertEqual(
-            continuous_computation_data['status_code'], 'idle')
-        self.assertTrue(continuous_computation_data['is_startable'])
-        self.assertFalse(continuous_computation_data['is_stoppable'])
+        self.assertEqual(len(continuous_computations_data), 1)
+        self.assertDictContainsSubset(
```

Comment:
You wrote:

> used assertDictContainsSubset as some fields cannot be asserted

in response to my question:

> Any reason why you can't replace all these lines with
> self.assertEqual(continuous_computations_data, ...) ?

This is not enough information. Could you please explain why "some fields cannot be asserted" and what efforts you have taken to try to be able to assert them? There should be a justification for every field that you are omitting.

Reply:
Sorry about that, I was thinking of some other test where it had fields like last_started or something similar containing time fields.

I changed this to assertEqual.